src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -346,6 +348,7 @@ export class SearchInterceptor {
       const { executionContext, strategy, ...searchOptions } = this.getSerializableOptions(options);
       return this.deps.http
         .post(`/internal/search/${strategy}${request.id ? `/${request.id}` : ''}`, {
```

Comment:
Is my assumption correct that `batchedFetch` will be handled in separate PR?

Reply:
Yes, that's correct.